<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="testCases.Validate_LostPassword" tests="11" time="39.404" ignored="8" failures="0" timestamp="17 Jul 2017 08:39:14 GMT" skipped="0" hostname="ehtesham-HP-ProBook-4540s" errors="1">
  <testcase name="testValidPasswordLength" time="0.319" classname="testCases.Validate_LostPassword"/>
  <testcase name="testEmailInvalid" time="31.007" classname="testCases.Validate_LostPassword">
    <error type="org.openqa.selenium.TimeoutException" message="Expected condition failed: waiting for element to be clickable: By.className: error-msg (tried for 30 second(s) with 500 MILLISECONDS interval)
Build info: version: &#039;3.1.0&#039;, revision: &#039;b94c902&#039;, time: &#039;2017-02-16 12:21:31 -0800&#039;
System info: host: &#039;ehtesham-HP-ProBook-4540s&#039;, ip: &#039;127.0.1.1&#039;, os.name: &#039;Linux&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;4.4.0-75-generic&#039;, java.version: &#039;1.8.0_131&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, mobileEmulationEnabled=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.30.477691 (6ee44a7247c639c0703f291d320bdf05c1531b57), userDataDir=/tmp/.org.chromium.Chromium.FTRSSU}, takesHeapSnapshot=true, pageLoadStrategy=normal, databaseEnabled=false, handlesAlerts=true, hasTouchScreen=false, version=59.0.3071.115, platform=LINUX, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=}]
Session ID: a9e35bc0427a11cbb50e3f9a864286a6">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for element to be clickable: By.className: error-msg (tried for 30 second(s) with 500 MILLISECONDS interval)
Build info: version: '3.1.0', revision: 'b94c902', time: '2017-02-16 12:21:31 -0800'
System info: host: 'ehtesham-HP-ProBook-4540s', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'amd64', os.version: '4.4.0-75-generic', java.version: '1.8.0_131'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, mobileEmulationEnabled=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.30.477691 (6ee44a7247c639c0703f291d320bdf05c1531b57), userDataDir=/tmp/.org.chromium.Chromium.FTRSSU}, takesHeapSnapshot=true, pageLoadStrategy=normal, databaseEnabled=false, handlesAlerts=true, hasTouchScreen=false, version=59.0.3071.115, platform=LINUX, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=true, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true, unexpectedAlertBehaviour=}]
Session ID: a9e35bc0427a11cbb50e3f9a864286a6
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:80)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:257)
	at utility.CommonFunctions.FindElementWithExplicitWait(CommonFunctions.java:32)
	at pageObjects.LostPassword_Page.errorEmail(LostPassword_Page.java:26)
	at testCases.Validate_LostPassword.testEmailInvalid(Validate_LostPassword.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </error>
  </testcase> <!-- testEmailInvalid -->
  <testcase name="testEmptyFields" time="1.314" classname="testCases.Validate_LostPassword"/>
  <testcase name="resetEmailPage" time="0.789" classname="testCases.Validate_LostPassword"/>
  <testcase name="testPasswordMatch" time="0.508" classname="testCases.Validate_LostPassword"/>
  <testcase name="testValidPasswordOverall" time="0.199" classname="testCases.Validate_LostPassword"/>
  <testcase name="setEmail" time="0.473" classname="testCases.Validate_LostPassword"/>
  <testcase name="setDOB" time="0.858" classname="testCases.Validate_LostPassword"/>
  <testcase name="linkRetry" time="0.634" classname="testCases.Validate_LostPassword"/>
  <testcase name="resetPassword" time="0.694" classname="testCases.Validate_LostPassword"/>
  <testcase name="A_navigateToLostPassword" time="2.609" classname="testCases.Validate_LostPassword"/>
  <testcase name="Login" classname="testCases.Validate_ResourcesPage">
    <skipped/>
  </testcase> <!-- Login -->
  <testcase name="loginMF" classname="testCases.Validate_ResourcesPage">
    <skipped/>
  </testcase> <!-- loginMF -->
  <testcase name="loginMX" classname="testCases.Validate_ResourcesPage">
    <skipped/>
  </testcase> <!-- loginMX -->
  <testcase name="testEmailEmpty" classname="testCases.Validate_LostPassword">
    <skipped/>
  </testcase> <!-- testEmailEmpty -->
  <testcase name="TestNotMoreThanPrevious" classname="testCases.Validate_SelfExclusionPage">
    <skipped/>
  </testcase> <!-- TestNotMoreThanPrevious -->
  <testcase name="Test_Limits_MaxAccountBalance" classname="testCases.Validate_Limits">
    <skipped/>
  </testcase> <!-- Test_Limits_MaxAccountBalance -->
  <testcase name="check_withdrawlock" classname="testCases.Validate_CashierPassword">
    <skipped/>
  </testcase> <!-- check_withdrawlock -->
  <testcase name="check_depositLock" classname="testCases.Validate_CashierPassword">
    <skipped/>
  </testcase> <!-- check_depositLock -->
</testsuite> <!-- testCases.Validate_LostPassword -->
